src/zap_hooks/requirements.txt

```diff
@@ -8,3 +8,4 @@ bleach
 selenium==4.14.0
 pyotp==2.6.0
 selenium-wire==5.1.0
+blinker==1.7.0
```

Comment:
We're on latest of selenium-wire. It has a blinker>=1.4 requirement. blinker 1.8.0 & /1.8.1, released 4/27 & 4/28, aren't compatible. Locking at latest blinker version before those.